### PR TITLE
feat: improve translation scripts

### PR DESCRIPTION
**Pull request by firebomb1**
## Description

Improves the maintenance scripts for translations.

- All the yarn commands have a `translations:` prefix:
  - `yarn translations:count-words-since` (previously: `yarn count-english-words-since`)
    - Supports using dates (ISO format) in addition of commit hashes
    - Supports passing an argument for the date/commit hash when calling the script
    - Counts new/modified strings separately
    - Offers a command to copy to invalidate modified strings (if any is present)
  - `yarn translations:invalidate-outdated` (previously: `yarn remove-outdated-translations`)
    - Supports using a date (ISO format) in addition of commit hashes
    - Supports passing an argument for the date/commit hash when calling the script
  - `yarn translations:update-path` (previously: `yarn update:translation-path`)
    - Content of the script unchanged (created by @gomesalexandre), only the way to call the script changed.
- Common functions moved to `scripts/translations/utils.ts`

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [x] :nail_care: New Feature (Breaking/Non-breaking Change)

## Risk
None. Scripts used internally by the Globalization WS.

### Engineering
None required, but welcome.

### Operations
None required.

## Screenshots (if applicable)
N/A

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -50,6 +47,9 @@
     "test:cypress:clean": "yarn cypress run --browser chrome --headless --env cleanMocks=true",
     "test:dev:web": "yarn react-app-rewired test --watch",
     "test:dev:packages": "yarn test:packages --watch",
+    "translations:count-words-since": "yarn ts-node --skipProject ./scripts/translations/countEnglishWordsSince.ts",
+    "translations:invalidate-outdated": "yarn ts-node --skipProject ./scripts/translations/invalidateOutdated.ts",
+    "translations:update-path": "yarn ts-node --skipProject ./scripts/translations/updateTranslationPath.ts",
```
Comment (0xApotheosis) [inline package.json:52]: Brilliant consolidation.

File: scripts/translations/invalidateOutdated.ts [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+process.chdir(repoPath)
+
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+})
+
+let commitId: string
+if (process.argv.length > 2) {
+  let commitId = process.argv.splice(2, process.argv.length - 1).join(' ')
```
Comment (0xApotheosis) [inline scripts/translations/invalidateOutdated.ts:25]: No need to redeclare.

```suggestion
  commitId = process.argv.splice(2, process.argv.length - 1).join(' ')
```

**Review by 0xApotheosis — APPROVED** [review]
👌

File: scripts/translations/invalidateOutdated.ts (hunk 2 above)
Comment (firebomb1) [reply scripts/translations/invalidateOutdated.ts:25]: Indeed, a bit of a careless copy/paste on my part from here: https://github.com/shapeshift/web/blob/00c817051c127d628ab5b51a661a02790351a1e7/scripts/translations/countEnglishWordsSince.ts#L21

Thanks for the review, I'll make the change in a new PR 😅

**Pull request merged**